frontend/src/components/navbar/Navbar.js

```diff
@@ -3,12 +3,13 @@ import Logo from "../../assets/Logo.jpeg";
 import CommonButton from "../button/commonButton/CommonButton";
 import { useAuth } from "../../contexts/UserContext";
```

Comment:
still imports are not in correct order in all the files

Reply:
okay, I have fixed it now